Mathlib/Data/ENat/Basic.lean

```diff
@@ -379,16 +379,16 @@ lemma mul_left_strictMono (ha : a ≠ 0) (h_top : a ≠ ⊤) : StrictMono (a * 
   rw [ENat.coe_lt_coe] at hxy
   exact Nat.mul_lt_mul_of_pos_left hxy (Nat.pos_of_ne_zero (by simpa using ha))
 
-lemma mul_right_strictMono (ha : a ≠ 0) (h_top : a ≠ ⊤) : StrictMono (· * a) := by
-  simpa [show (· * a) = (a * ·) by simp [mul_comm]] using mul_left_strictMono ha h_top
+lemma mul_left_strictMono (ha : a ≠ 0) (h_top : a ≠ ⊤) : StrictMono (· * a) := by
+  simpa [show (· * a) = (a * ·) by simp [mul_comm]] using mul_right_strictMono ha h_top
```

Comment:
Arguably this is out of the scope of this PR, but does this work?
```suggestion
  simpa [mul_comm] using mul_right_strictMono ha h_top
```

Reply:
golfed another way instead